docs/src/examples/components/Dropdown/Usage/DropdownExampleControlled.shorthand.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react'
+import { Dropdown, Flex, Text } from '@stardust-ui/react'
+
+const inputItems = ['Bruce Wayne', 'Natasha Romanoff', 'Steven Strange', 'Alfred Pennyworth']
+
+class DropdownExampleControlled extends React.Component {
+  state = { open: false }
+
+  render() {
+    const open = this.state.open
+    return (
+      <Flex gap="gap.large" vAlign="center">
+        <Dropdown
+          open={open}
+          onOpenChange={this.handleOpenChange}
+          items={inputItems}
+          placeholder="Select your hero"
+        />
+        <Text weight="semibold" content={`Dropdown open state is: "${open}"`} />
```

Comment:
We should use knobs  for `open` there actually.

Reply:
how exactly?

If I do something like this:

```typescript
import * as React from 'react'
import { Dropdown } from '@stardust-ui/react'
import { DropdownExampleControlledKnobsProps } from './DropdownExampleControlled.knobs'

const inputItems = ['Bruce Wayne', 'Natasha Romanoff', 'Steven Strange', 'Alfred Pennyworth']

const DropdownExampleControlled: React.FC<{ knobs: DropdownExampleControlledKnobsProps }> = ({ knobs }) => (
  <Dropdown
    open={knobs.open}
    onOpenChange={() => {}} // redundant
    items={inputItems}
    placeholder="Select your hero"
  />
)

export default DropdownExampleControlled
```
`onOpenChange` prop becomes redundant because `open` flag is controlled by the knob now. I'd still like to keep the meaning of the example and showcase both `open` and `onOpenChange` and I saw this pattern being used for:

- [Popup in controlled mode](https://stardust-ui.github.io/react/components/popup#types-popup-controlled-example)
- [List in controlled mode](https://stardust-ui.github.io/react/components/list#types-selectable-controlled)

@layershifter what do you think?

Reply:
Agree, let's leave it as is.

Reply:
cool, resolving